indico/core/auth.py

```diff
@@ -39,6 +40,30 @@ def default_group_provider(self):
         return next((p for p in self.identity_providers.itervalues()
                      if p.supports_groups and p.settings.get('default_group_provider')), None)
 
+    @property
+    def sync_provider(self):
+        """The synchronization provider.
+
+        This is the identity provider used to sync user data.
+        """
+        return next((p for p in self.identity_providers.itervalues()
+                     if p.supports_refresh and p.settings.get('synced_fields')), None)
+
+    @property
+    def synced_fields(self):
+        """The keys to be synchronized
```

Comment:
Well keys refer to application keys, or the keys of the user. They are called keys as in multipass. For synchronization however we refer to them as fields... I  like fields better but I think we should unify the terminonology everywhere.